proof/invariant-abstract/DetSchedAux_AI.thy

```diff
@@ -562,12 +562,6 @@ lemma retype_region_valid_blocked[wp]:
   apply (blast intro: valid_blocked_fold_update)
   done
 
-=======
-crunch create_cap, cap_insert
-  for valid_queues[wp]: valid_queues
-  (wp: valid_queues_lift)
->>>>>>> 3667ef81cd60b2b814cd28e59231bc7399250fc6
```

Comment:
do we want merge markers in commits? this might be worth squashing back into the merge, if that's even possible

Reply:
Yeah, this was a mistake. It should be possible to squash it into the merge commit but it's always fiddly to do so, which is why I left in this and the other obvious mistakes I made while resolving conflicts and fixed them in the next commit. I'll give it a go but will leave it if I run into problems.